pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -92,6 +92,69 @@ type ClusterServiceBrokerSpec struct {
 	// can be manually incremented by a user to manually trigger a relist.
 	// +optional
 	RelistRequests int64 `json:"relistRequests"`
+
+	// CatalogRestrictions allows adding restrictions onto Class/Plans on relist.
+	// +optional
+	CatalogRestrictions *ServiceClassCatalogRestrictions `json:"catalogRestrictions,omitempty"`
```

Comment:
Why not just make these arrays instead of pointer to typedef?

Reply:
ServiceClassCatalogRestrictions is a struct that has arrays for class and plan. So this is a pointer to a struct, not a pointer to a typedef'ed array.

Reply:
Now looking at it, I did not mean for this to contain "Class", this type is suppose to be ServiceCatalogRestrictions

I will update.

Reply:
Updated.